From 4fdac66777dd780bd1e46d91bf38513832934695 Mon Sep 17 00:00:00 2001
From: Mark Reynolds <mreynolds@redhat.com>
Date: Thu, 6 Aug 2015 10:13:40 -0400
Subject: [PATCH 35/39] Ticket 48215 - update dbverify usage

Description:  Need to add the "-a" argument usage

https://fedorahosted.org/389/ticket/48215
(cherry picked from commit 20284e6539f557efc0679d974d5156cdcd55c407)
(cherry picked from commit 8e08c8b53641d807b63d87ee79564c596c5da4dd)
---
 ldap/admin/src/scripts/dbverify.in | 15 ++++++++-------
 1 file changed, 8 insertions(+), 7 deletions(-)

diff --git a/ldap/admin/src/scripts/dbverify.in b/ldap/admin/src/scripts/dbverify.in
index 778a9ba..461cc16 100755
--- a/ldap/admin/src/scripts/dbverify.in
+++ b/ldap/admin/src/scripts/dbverify.in
@@ -14,15 +14,16 @@ PATH=$PATH:/bin
 
 usage()
 {
-    echo "Usage: dbverify [-Z serverID] [-n backend_instance] [-V] [-v] [-d debuglevel] [-h]"
+    echo "Usage: dbverify [-Z serverID] [-n backend_instance] [-a db_directory ] [-V] [-v] [-d debuglevel] [-h]"
     echo "Note if \"-n backend\" is not passed, verify all DBs."
     echo "Options:"
-    echo "        -Z             - Server instance identifier"
-    echo "        -n backend     - Backend database name.  Example: userRoot"
-    echo "        -V             - Verbose output"
-    echo "        -d debuglevel  - Debugging level"
-    echo "        -v             - Display version"
-    echo "        -h             - Display usage"
+    echo "        -Z              - Server instance identifier"
+    echo "        -n backend      - Backend database name.  Example: userRoot"
+    echo "        -a db_directory - Database directory"
+    echo "        -V              - Verbose output"
+    echo "        -d debuglevel   - Debugging level"
+    echo "        -v              - Display version"
+    echo "        -h              - Display usage"
 }
 
 display_version="no"
-- 
1.9.3

